CMakeLists.txt

```diff
@@ -4,10 +4,10 @@ project(yapvm)
 
 set(CMAKE_CXX_STANDARD 23)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+//set(CMAKE_CXX_FLAGS "-stdlib=libc++")
```

Comment:
It's your local change, it breaks compatibility with windows when building with MSVC in example